src/ldap_driver.c

```diff
@@ -867,7 +867,8 @@ static dns_dbmethods_t ldapdb_methods = {
 	findext,
 	setcachestats,
 	hashsize,
-	nodefullname
+	nodefullname,
+	NULL, // getsize method not implemented (related BZ1353563)
```

Comment:
I haven't check but based on mail from freeipa-devel[1] I would say that you mixed tabs and spaces

[1] https://lists.fedorahosted.org/archives/list/freeipa-devel@lists.fedorahosted.org/message/KPLKN6SHT3OVDL7LMU273A7GGASIPM45/attachment/2/bind-dyndb-ldap-pr-18.patch